<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insert_substring</class>
 <widget class="QDialog" name="insert_substring">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert substring</string>
  </property>
  <widget class="QLineEdit" name="line_val">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Line:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="character_val">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Character:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="substring_val">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Substring:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>insert_substring</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
